string name
string ID
Vector4 relativeHeading
float32 range
float32 angularRange
uint16 targetSticker
float32[] data
